cmake/secp256k1.cmake

```diff
@@ -57,3 +57,4 @@ if(MSVC)
 endif()
 
 target_link_libraries(secp256k1 PRIVATE core_base_interface)
+set_target_properties(secp256k1 PROPERTIES EXPORT_COMPILE_COMMANDS OFF)
```

Comment:
What will happen here when we switch to using secp's CMake build?

Reply:
> What will happen here when we switch to using secp's CMake build?

I can see two option.

1. We keep settling target property externally as we currently do. 

Or

2. Upstream secp build system can force this property disabled when being a subproject.